app/src/services/census/index.ts

```diff
@@ -1,21 +1,21 @@
 import {ContainerModule} from 'inversify';
 import ServiceInterface, {SERVICE} from '../../interfaces/ServiceInterface';
 import config from '../../config';
-import PS2EventClient from 'ps2census/dist/client/Client'; // TODO: Await microwave's type fixes
 import CensusStreamService from './CensusStreamService';
-import {PS2ClientConfig} from 'ps2census/dist/client/utils/Types';
+import Client from 'ps2census/dist/client/Client';
+import {EventStreamManagerConfig} from 'ps2census/dist/client/utils/Types';
 
```

Comment:
@microwavekonijn isn't the client exposed from the root of your package?

Reply:
My bad, client is available but EventStreamManagerConfig is not exported

Reply:
@microwavekonijn making you aware certain types aren't exposed

Reply:
Oh, shoot forgot to export that file completely.